udata/core/dataset/views.py

```diff
@@ -86,8 +88,86 @@ def get_context(self):
         context['can_edit'] = DatasetEditPermission(self.dataset)
         context['can_edit_resource'] = ResourceEditPermission
         context['discussions'] = Discussion.objects(subject=self.dataset)
+
         return context
 
+    def get_json_ld(self):
```

Comment:
I'm not sure the views should host the JSON-LD construction as this is related to models.